zerver/tests/test_push_notifications.py

```diff
@@ -428,6 +429,68 @@ def test_send_notification_endpoint(self) -> None:
             remote=server,
         )
 
+    def test_old_timestamp_format(self) -> None:
+        hamlet = self.example_user("hamlet")
+        RemotePushDeviceToken.objects.create(
+            kind=RemotePushDeviceToken.GCM,
+            token=hex_to_b64("aaaaaa"),
+            user_id=hamlet.id,
+            server=RemoteZulipServer.objects.get(uuid=self.server_uuid),
+        )
+
+        time_sent = now().replace(microsecond=0)
+        with time_machine.travel(time_sent):
```

Comment:
@alexmv I was just git grepping through `time_machine.travel` calls that we have without `tick=False` and bumped into this one as the most recent we have like that. Is it intentional here or should we be fixing this to add `tick=False`? (and probably the other instances too)

Reply:
Yeah, this needs a `tick=False`.  Thanks for catching that!

Reply:
Cool, I'll open a PR

Reply:
Part of this got pulled into #26940.